Remotion/Web/Core/Res/HTML/SmartPage.js

```diff
@@ -1178,6 +1178,8 @@ function SmartPage_Context(
       _submitState = submitState;
     else if (_submitState.IsAutoPostback && !isAutoPostback)
       _submitState.NextSubmitState = submitState;
+    else
+      _submitState.NextSubmitState = { CancelSubmit: true };

```

Comment:
I'm thinking, for consistency reasons, we should add CancelSubmit=false to line 1169 and Submitter=null to line 1182. This should make it more explicit that we have considered the relevant attributes and decided to make them null/false.

Please do, re-check the double postback catch behavior and the BocList bugfix, then we should be done.

Reply:
Tested it, works as intended.